### PR TITLE
Added support for opening folders in a new tab by clicking the middle mouse button

### LINKED ISSUE #1343: Middle-click open in new tab
**Is your feature request related to a problem? Please describe.**
Not really a problem. It's just nice to have.

**Describe the solution you'd like**
Middle-click on a sidebar item or on a folder opens it in a new tab.

**Describe alternatives you've considered**
None.

**Additional context**
None.

### PR DESCRIPTION
**Pull request by yaichenbaum**
Fixes #1343

This works in all the layout modes

### CODE REVIEW

File: Files/Interacts/Interaction.cs [hunk 1]
```diff
@@ -145,6 +145,23 @@ public async void OpenDirectoryInNewTab_Click()
...
 
+        public async void ItemPointerPressed(object sender, PointerRoutedEventArgs e)
+        {
+            if (e.GetCurrentPoint(null).Properties.IsMiddleButtonPressed)
+            {
+                if ((e.OriginalSource as FrameworkElement)?.DataContext is ListedItem Item && Item.PrimaryItemAttribute == StorageItemTypes.Folder)
+                {
+                    if (Item.IsShortcutItem)
+                    {
+                        await MainPage.AddNewTabByPathAsync(typeof(ModernShellPage), ((e.OriginalSource as FrameworkElement)?.DataContext as ShortcutItem)?.TargetPath ?? Item.ItemPath);
```
Comment (tsvietOK) [inline Files/Interacts/Interaction.cs:156]:
```suggestion
                        OpenPathInNewTab(((e.OriginalSource as FrameworkElement)?.DataContext as ShortcutItem)?.TargetPath ?? Item.ItemPath);
```

File: Files/Interacts/Interaction.cs [hunk 2]
```diff
@@ -145,6 +145,23 @@ public async void OpenDirectoryInNewTab_Click()
             }
         }
 
+        public async void ItemPointerPressed(object sender, PointerRoutedEventArgs e)
```
Comment (tsvietOK) [inline Files/Interacts/Interaction.cs:148]:
```suggestion
        public void ItemPointerPressed(object sender, PointerRoutedEventArgs e)
```

Comment (yaichenbaum) [reply Files/Interacts/Interaction.cs:148]: There is an `Await` so that's required.

Comment (tsvietOK) [reply Files/Interacts/Interaction.cs:148]: It is not, if you accept first suggestion.

**Review by tsvietOK — APPROVED** [review]

**Pull request merged**